src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/services/citizen/SettlementReachedCitizenNotificationServiceTest.java

```diff
@@ -0,0 +1,174 @@
+package uk.gov.hmcts.cmc.claimstore.controllers.services.citizen;
+
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.ResultActions;
+import uk.gov.hmcts.cmc.claimstore.BaseOfferTest;
+import uk.gov.hmcts.cmc.domain.models.offers.MadeBy;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static uk.gov.hmcts.cmc.claimstore.controllers.PathPatterns.CLAIM_REFERENCE_PATTERN;
+
+public class SettlementReachedCitizenNotificationServiceTest extends BaseOfferTest {
```

Comment:
Please rename this to be `SettlementReachedCitizenNotificationTest` - simply remove `Service` word from it.